kubernetes_state/datadog_checks/kubernetes_state/kubernetes_state.py

```diff
@@ -766,3 +802,33 @@ def count_objects_by_tags(self, metric, scraper_config):
 
         for tags, count in iteritems(object_counter):
             self.gauge(metric_name, count, tags=list(tags))
+
+    def _build_tags(self, label_name, label_value, scraper_config, hostname=None):
+        """
+        Build a list of formated tags from `label_name` parameter. It also depend of the
+        check configuration ('keep_ksm_labels' parameter)
+        """
+        _tags = list()
+        # first use the labels_mapper
+        tag_name = scraper_config['labels_mapper'].get(label_name, label_name)
+        # then try to use the kube_labels_mapper
+        kube_tag_name = kube_labels_mapper.get(tag_name, tag_name)
+        label_value = to_string(label_value).lower()
+        _tags.append('{}:{}'.format(to_string(kube_tag_name), label_value))
+        if self.keep_ksm_labels and (kube_tag_name != tag_name):
+            _tags.append('{}:{}'.format(to_string(tag_name), label_value))
+        return _tags
+
+    def _metric_tags(self, metric_name, val, sample, scraper_config, hostname=None):
+        """
+        Redefine this method to allow labels duplication, during migration phase
+        """
+        custom_tags = scraper_config['custom_tags']
+        _tags = list(custom_tags)
+        _tags.extend(scraper_config['_metric_tags'])
```

Comment:
Note that you use `+=` almost everywhere else. I tend to prefer extend in general, but we should be consistent.